src/module-elasticsuite-core/etc/adminhtml/system.xml

```diff
@@ -180,6 +180,48 @@
                     <comment><![CDATA[Maximum number of popular search terms to display in autocomplete results.]]></comment>
                 </field>
             </group>
+
+            <group id="advanced" translate="label" type="text" sortOrder="99" showInDefault="1" showInWebsite="1" showInStore="1">
+                <label>Advanced Settings</label>
+                <field id="extension_enabled" translate="label comment" type="select" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
+                    <label>Use suggested search terms to fetch results</label>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                    <comment><![CDATA[Default: Yes. When enabled, autocompletion box results will be computed from the suggested search terms. This is intended to suggest results when the user is in the process of typing. Eg : When the user is typing "comp", the popular search term "computer" will be suggested, and therefore products or categories matching "computer" will be shown.]]></comment>
```

Comment:
First "sentence" should be "Default: Yes (legacy behavior)".
I would change the rest for clarity ("autocompletion box results" is a bit vague) to something like : 
"When set to Yes, products and category autocomplete will use the search terms suggested in the popular term autocomplete. This is intended to help find results when the user is in the process of typing a word." + 
(untouched) "Eg : When the user is typing "comp", the popular search term "computer" will be suggested, and therefore products or categories matching "computer" will be shown."